articles/storage/files/storage-files-migration-robocopy.md

```diff
@@ -215,6 +215,18 @@ You should be prepared to run multiple rounds of RoboCopy against a given namesp
 
 `/R:5 /W:5` is a reasonable setting that you can adjust to your liking. In this example, a failed file will be retried five times, with five-second wait time between retries. If the file still fails to copy, the next RoboCopy job will try again. Often files that failed because they are in use or because of timeout issues might eventually be copied successfully this way.
 
+### Estimating storage transaction charges
+
+As you begin your migration into Azure, RoboCopy copies your files and folders into your target Azure Storage locations. Depending on the configuration of these storage targets, usage charges may apply.
+
+If you've determined that your Azure storage product charges for transactions, it may be difficult to estimate the number generated by your migration.
+
+- It's not possible to estimate the number of transactions based on the utilized storage capacity of the source. The number of transactions scales with the number of namespace items (files and folder) and their properties that are migrated, not their size. For example, more transactions are required to migrate 1 GiB of small files than 1 GiB of larger files.
+- In order to minimize downtime, you may need to run copy operations several times between a source and its target. All source and target items are processed during each copy operation, though subsequent runs finish faster. After the initial operations, only the differences introduced between copy runs are transported over the network. It's important to understand that although less data is being transported, the number of transactions required may remain the same.
+- Copying the same file twice might not result in the same number of transactions. Processing an item migrated in a previous copy run may result in only a few read transactions. In contrast, changes to metadata or content between copy runs may require a larger number of transactions to update the target. Each file in your namespace may have unique requirements, resulting in a different number of transactions.
+
+It's advisable to run some initial tests on your own data, to better understand how many transactions are incurred. From here, you can get a better idea of the total number of transactions a file migration may generate. 
```

Comment:
```suggestion
It's advisable to run some initial tests on your own data to better understand how many transactions are incurred. This will give you a better idea of the total number of transactions a file migration might generate. 
```